### ISSUE TITLE
Data fields for Google Lat/Lon

### ISSUE THREAD

**Issue by djireland** [matthewproctor/australianpostcodes#30]
The data fields table mentions "Google Lat/Lon" but the corresponding fields in the (JSON) data  don't use this convention.

 Is locations from Google API denoted lat/lon_precise?

**Comment by milkmanmatty** [comment]
That's correct [djireland](https://github.com/djireland), the google provided fields are called `Long_precise`, `Lat_precise`. However I would very careful using them as in some cases they are actually inaccurate - even by Google's standards.

For example even just looking at Aus Capital Cities, `Darwin 0800` has a Lat, Lon of  `-12.458684, 130.83668` which looks correct [according to google maps](https://www.google.com/maps/place/12%C2%B027'31.3%22S+130%C2%B050'12.1%22E/@-12.4497169,130.826131,12.51z/data=!4m5!3m4!1s0x0:0xef8789e5fbd170c!8m2!3d-12.458684!4d130.83668).
Going by the `Long_precise`, `Lat_precise` for the same entry gives a sightly different position of `-12.3932794, 130.7766611` which ends up [way in the water](https://www.google.com/maps/place/12%C2%B023'35.8%22S+130%C2%B046'36.0%22E/@-12.388162,130.790052,12.53z/data=!4m5!3m4!1s0x0:0x9448c3de855abb98!8m2!3d-12.39328!4d130.77666) - this is clearly incorrect!

So fair warning, I've recently been stung by this for various geo locations and will be sticking with the "non-precise" lat lon coordinates.
